packages/story-editor/src/components/panels/design/pageBackgroundAudio/test/backgroundAudio.js

```diff
@@ -175,8 +175,7 @@ describe('BackgroundAudioPanel', () => {
         loop: true,
       },
     });
-    const input = screen.getByRole('textbox', { name: 'Filename' });
-    expect(input).toHaveValue('track.vtt');
+    expect(screen.getByText('track.vtt')).toBeInTheDocument();
```

Comment:
References to `trackName` should be removed.

Reply:
What do you mean?

Reply:
In this test file, there are reference to trackName in data.